docs/sources/setup/install/helm/deployment-guides/aws.md

```diff
@@ -0,0 +1,763 @@
+  service:
+    type: LoadBalancer
+  basicAuth: 
+      enabled: true
+      existingSecret: loki-basic-auth
+
+
+singleBinary:
+  replicas: 3
+  persistence:
+    storageClass: gp2
+    accessModes:
+      - ReadWriteOnce
+    size: 30Gi
+
+# Zero out replica counts of other deployment modes
+backend:
+  replicas: 0
+read:
+  replicas: 0
+write:
+  replicas: 0
+
+ingester:
+  replicas: 0
+querier:
+  replicas: 0
+queryFrontend:
+  replicas: 0
+queryScheduler:
+  replicas: 0
+distributor:
+  replicas: 0
+compactor:
+  replicas: 0
+indexGateway:
+  replicas: 0
+bloomCompactor:
+  replicas: 0
+bloomGateway:
+  replicas: 0
+
+```
+
+{{< /collapse >}}
+
+{{< admonition type="caution" >}}
+Make sure to replace the placeholders with your actual values.
+{{< /admonition >}}
+
+It is critical to define a valid `values.yaml` file for the Loki deployment. To remove the risk of misconfiguration, let's break down the configuration options to keep in mind when deploying to AWS:
+
+- **Loki Config vs. Values Config:**
+  - The `values.yaml` file contains a section called `loki`, which contains a direct representation of the Loki configuration file.
+  - This section defines the Loki configuration, including the schema, storage, and querier configuration.
+  - The key configuration to focus on for chunks is the `storage_config` section, where you define the S3 bucket region and name. This tells Loki where to store the chunks.
+  - The `ruler` section defines the configuration for the ruler, including the S3 bucket region and name. This tells Loki where to store the alert and recording rules.
+  - For the full Loki configuration, refer to the [Loki Configuration]({{< relref "../../../../configure" >}}) documentation.
+
+- **Storage:**
+  - Defines where the Helm chart stores data.
+  - Set the type to `s3` since we are using Amazon S3.
+  - Configure the bucket names for the chunks and ruler to match the buckets created earlier.
+  - The `s3` section specifies the region of the bucket.
+
+- **Service Account:**
+  - The `serviceAccount` section is used to define the IAM role for the Loki service account.
+  - This is where the IAM role created earlier is linked.
+
+- **Gateway:**
+  - Defines how the Loki gateway will be exposed.
+  - We are using a `LoadBalancer` service type in this configuration.
+
+
+### Deploy Loki
+
+Now that you have created the `values.yaml` file, you can deploy Loki using the Helm chart.
+
+1. Deploy using the newly created `values.yaml` file:
+
+    ```bash
+    helm install --values values.yaml loki grafana/loki -n loki --create-namespace
+    ```
+    **It is important to create a namespace called `loki` as our trust policy is set to allow the IAM role to be used by the `loki` service account in the `loki` namespace. This is configurable but make sure to update your service account**
+
+1. Verify the deployment:
+
+    ```bash
+    kubectl get pods -n loki
+    ```
+    You should see the Loki pods running.
+
+    ```console
```

Comment:
You should clarify if this output is for the monolithic or microservices deployment (above).